libs/response-lib.js

```diff
@@ -14,11 +14,22 @@ export function resourceNotFound(body) {
   return buildResponse(404, body);
 }
 
-function buildResponse(statusCode, body) {
+export function redirect(location) {
+  return buildResponse(301, null, { Location: location });
+}
+
+export function redirectWithError(location, error, errorMsg) {
+  const url = `${location}?error=${error}&error_description=${errorMsg}`;
+  return redirect(url);
+}
+
+function buildResponse(statusCode, body, headers = {}) {
   return {
     statusCode,
     headers: {
-      'Access-Control-Allow-Origin': '*'
+      'Access-Control-Allow-Origin': '*',
```

Comment:
Just for my knowledge: I would expect `Access-Control-Allow-Origin` to be a GateWay setting. Is it correct to manage CORS from the lambda code?

Reply:
This is just so the lambda can communicate with API Gateway. The gateway will take care of this. I am not an expect :D, this is the recommended setting from the Serverless Framework when you have API Gateway and Cognito